homeassistant/components/camera/push.py

```diff
@@ -50,7 +55,8 @@
 
     cameras = [PushCamera(config[CONF_NAME],
                           config[CONF_BUFFER_SIZE],
-                          config[CONF_TIMEOUT])]
+                          config[CONF_TIMEOUT],
+                          config[CONF_TOKEN])]
```

Comment:
This expects the CONF_TOKEN to be always set, which is not the case as it can be optional.